metrics.go

```diff
@@ -69,6 +69,14 @@ var (
 		},
 		nil,
 	)
+	metricDeviceTotalCapacityBytes = prometheus.NewDesc(
+		"smartctl_device_total_capacity_bytes",
```

Comment:
Is there a better name for the metric? Having "total" in the name is confusing when the [naming convention](https://prometheus.io/docs/instrumenting/writing_exporters/#naming) uses `_total` as a suffix for a counter. What about "smartctl_device_nvme_capacity_bytes"? Other metric names have "scsi" so I don't see why we can't use "nvme".


> `_total` is a convention for counters, you should use it if you’re using the COUNTER type.

Reply:
Renamed